base/ghga-common/templates/_kafka-topics.tpl

```diff
@@ -70,12 +70,14 @@ spec:
     {{- end }}
     {{- include "common.tplvalues.render" (dict "value" $topicsACL "context" $) | nindent 4 }}
     {{- end }}
-    - operations:
-      - All
-      resource:
-        name: '*'
-        patternType: literal
-        type: group
+    {{- with .Values._consumer_group -}}
+    {{- $consumerGroupACL := list -}}
+    {{- $aclEntry := hasKey . "operations" | ternary . (dict "operations" (list "Read")) -}}
+    {{- $aclEntry = hasKey . "resource" | ternary (merge $aclEntry .) (merge $aclEntry (dict "resource" (dict "patternType" "literal" "type" "group"))) -}}
+    {{- $aclEntry = set $aclEntry "resource" (merge $aclEntry.resource (dict "name" (print $.Values.serviceName "-consumer"))) -}}
```

Comment:
Is that intentionally in a new statement?

Reply:
Nope, good catch